sass/responsive/_desktop.scss

```diff
@@ -1,5 +1,34 @@
 @charset 'UTF-8';
 
+@media screen and (min-width: 1680px) {
+    .sidebar--right {
+        width: 500px;
+        @include translateX(100%);
+
+        &.move--left {
+            @include translateX(0);
+        }
+    }
+
+    .inner-wrap {
+        &.move--left {
+            .search__form {
+                width: 260px;
+            }
+
+            .app__content {
+                > #post-list {
+                    margin-right: 500px !important;
+                }
```

Comment:
I would stay away from ```!important``` and look into the selector specificity in order to make this work correctly.